src/runtime/contrib/tensorrt/tensorrt_ops.h

```diff
@@ -953,29 +1019,210 @@ class ResizeOpConverter : public TrtOpConverter {
   void Convert(AddTrtLayerParams* params) const {
     auto input = params->inputs.at(0).tensor;
     const auto* attrs = params->call->attrs.as<ResizeAttrs>();
-    static const std::unordered_map<std::string, nvinfer1::ResizeMode>
-        op_map = {
-            {"nearest_neighbor", nvinfer1::ResizeMode::kNEAREST},
-            {"bilinear", nvinfer1::ResizeMode::kLINEAR}};
+    static const std::unordered_map<std::string, nvinfer1::ResizeMode> op_map =
+        {{"nearest_neighbor", nvinfer1::ResizeMode::kNEAREST},
+         {"bilinear", nvinfer1::ResizeMode::kLINEAR}};
     auto it = op_map.find(attrs->method);
     CHECK(it != op_map.end()) << "Unsupported resize type " << attrs->method;
     CHECK_EQ(attrs->size.size(), 2);
     auto output_dims = TrtDimsToVector(input->getDimensions());
-    CHECK_EQ(output_dims.size(), 3);
+    const int required_rank =
+        params->network->hasImplicitBatchDimension() ? 3 : 4;
+    CHECK_EQ(output_dims.size(), required_rank);
+    CHECK(attrs->layout == "NCHW" || attrs->layout == "NHWC");
+    int h_index = attrs->layout == "NCHW" ? 2 : 1;
+    int w_index = attrs->layout == "NCHW" ? 3 : 2;
+    if (params->network->hasImplicitBatchDimension()) {
+      h_index -= 1;
+      w_index -= 1;
+    }
+    output_dims[h_index] = attrs->size[0].as<IntImmNode>()->value;
+    output_dims[w_index] = attrs->size[1].as<IntImmNode>()->value;
+
+    nvinfer1::IResizeLayer* resize_layer = params->network->addResize(*input);
+    CHECK(resize_layer != nullptr);
+    resize_layer->setResizeMode(it->second);
+    resize_layer->setOutputDimensions(VectorToTrtDims(output_dims));
+    resize_layer->setAlignCorners(attrs->coordinate_transformation_mode ==
+                                  "align_corners");
+    params->outputs.push_back(resize_layer->getOutput(0));
+  }
+};
+#endif  // TRT_VERSION_GE(6, 0, 1)
+
+#if 0 && TRT_VERSION_GE(5, 1, 5)
```

Comment:
just remove this?

Reply:
Thanks, removed all TRT NMS related code